pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -85,6 +127,23 @@ func (p *ConfigurationEditFlags) Apply(service *servingv1alpha1.Service, cmd *co
 		return err
 	}
 
+	name := ""
```

Comment:
This `apply()` method is already quite a mouthful could this revision name handling block be extracted in an own method e.g. `setRevisionName(cmd)` ?

Reply:
Done